extensions/Khronos/KHR_mesh_compression/README.md

```diff
@@ -63,60 +64,53 @@ The extension then could be used like:
             "mode" : 4
             "extensions" : {
                 "KHR_draco_geometry_compression" : {
+                    "bufferView" : 5,
                     "buffer" : 10,
                     "byteOffset" : 1024,
                     "byteLength" : 10000,
```

Comment:
Looks like `buffer`, `byteOffset`, and `byteLength` should be removed from example.